### ISSUE TITLE
pushid'yi nereden bulacağız?

### ISSUE THREAD

**Issue by jwaxy** [insanolanbiri/eokulapi#1]
pushid generator'u implemente etmemişsin peki o değeri nasıl alacağız? Uygulamada giriş yaparken arkadakai requestleri mi dinlememiz gerekiyor?

bir de eline sağlık çok güzel bir projeye benziyor fakat pek ilgi görmemiş gibi :D

**Comment by insanolanbiri** [comment]
> Uygulamada giriş yaparken arkadakai requestleri mi dinlememiz gerekiyor?

Evet, ben eokul android uygulamasını apk olarak indirip [reflutter](https://github.com/Impact-I/reFlutter) ile yamalamıştım. Öğrenci eklemesini yaptıktan sonra proxyle alacağınız uid değerini burada kullanabiliyorsunuz.

Pushidnin nasıl elde edildiğini henüz çözebilmiş değilim.

**Comment by onurback** [comment]
Size ulaşabileceğim bir iletişim adresiniz var mı? Eren Akgün
